### ISSUE TITLE
Keep comments when commenting a block of code

### ISSUE THREAD

**Issue by sxywu** [observablehq/feedback#89]
**Is your feature request related to a problem? Please describe.**
When I want to comment out a block of code, the lines that are already commented out (notes for myself, description of the function, etc.) gets uncommented.  This is frustrating when I want to uncomment the whole block, the original comment lines are also uncommented and I'm left with "Unexpected token" errors and have to manually go back in and comment those lines again.

**Describe the solution you'd like**
When I go to comment a whole block, the commented lines are kept:
```
// // some comment
// rest of commented out code
```

**Comment by jrus** [comment]
This is a regression from Classic -> Next code editor views.

**Comment by wiltsecarpenter** [comment]
Yes, this should be fixed now. Please let us know if it is not working for you.
